### PR TITLE
text added to connections

### PR DESCRIPTION
**Pull request by corinnebosley**

### CODE REVIEW

File: doc/write_your_own/connections/filePickerSimple.ipynb [hunk 1]
```diff
@@ -56,7 +56,7 @@
   },
```
Comment (annabooton) [inline doc/write_your_own/connections/filePickerSimple.ipynb]: This demonstrates how to "select multiple files" yet the result in the last line only contains one file?

File: doc/write_your_own/connections/intro.rst [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
 
-Here are some examples of how you would connect them:
+For widgets which possess hard coded options, the only elements you need are the list of input options and the widget itself.
+Widgets that depend on the response of previous selections need:
+    - An initial list of input options
+    - The widget itself
+    - A function defining what change occurs if a selection occurs in an influencing widget
+    - A statement to observe the influencing widget and execute the function upon event of a change.
+
+While you are adding these elements to your notebook, you will also benefit from moving all your widgets into a single cell so that they can communicate with each other without having to re-run.
```
Comment (dkillick) [inline doc/write_your_own/connections/intro.rst]: This is in a different tense to other elements of the docs. You could also do with specifying what you're planning to re-run at the end of the sentence! How about:

"When adding these elements... a single cell. Doing so means they can... without having to re-run all the cells that contain widgets."

Note: it's not clear what 'these' in the above refers to. Are they connecting elements? Are they widgets that depend on the response of previous selections?

File: doc/write_your_own/connections/intro.rst [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
+When you have done this, you can create containers to display groups of widgets instead of having to display each one individually.
```
Comment (dkillick) [inline doc/write_your_own/connections/intro.rst]: What's a container?

**Comment by corinnebosley** [comment]
@dkillick @annabooton I have made the changes that Pete suggested but this is not yet Anna-approved.

**Comment by corinnebosley** [comment]
@dkillick @marqh 
I am happy with this.

**Comment by corinnebosley** [comment]
@dkillick

File: doc/write_your_own/connections/intro.rst [hunk 4]
```diff
@@ -1,18 +1,38 @@
...
 Links to Notebooks:
 -------------------
 
-filePickerSimple.link
-
-interactions.link
+:download:`Simple File Picker <filePickerSimple.ipynb>`
 
-interactions2.link
+:download:`Interactions <interactions_1.ipynb>`
```
Comment (dkillick) [inline doc/write_your_own/connections/intro.rst]: I do wish it wasn't called `interactions_1`...

**Comment by dkillick** [comment]
Good stuff good stuff good stuff!

**Comment by dkillick** [comment]
In it goes! Well done all

**Pull request merged**